lib/Api/V1Api.php

```diff
@@ -9,12 +9,13 @@
 
 class V1Api {
 	private RowService $rowService;
-
 	private ColumnService $columnService;
+	private string $userId;
 
-	public function __construct(ColumnService $columnService, RowService $rowService) {
+	public function __construct(ColumnService $columnService, RowService $rowService, string $userId) {
```

Comment:
```suggestion
	public function __construct(ColumnService $columnService, RowService $rowService, ?string $userId) {
```

Dependency injection might pass in null, so this should be allowed to avoid errors in the logs

Reply:
You might want to instead check and throw an error in places where it is used